### ISSUE TITLE
Tray icon disappears unexpectedly

### ISSUE THREAD

**Issue by JnLlnd** [sabrogden/Ditto#101]
Hi,

I'm running DittoSetup_64bit_3_24_184_0. Sometimes (I would even say frequently), the Ditto icon in the Tray disappears unexpectedly. Ditto is still running and its menu remains available using the keyboard shortcut but I can't open it from the taskbar (and access the right-click menu). I need to restart the app to restore its icon.

Jean

**Comment by sabrogden** [comment]
I have had reports of this happening off and on for a long time.  Pretty sure this is some windows issue causing the icon to not show.  Windows 10?  You can always access Ditto by the hot key, ctrl-` by default (tilde key, just below escape) or this can be changed in options.

**Comment by JnLlnd** [comment]
> Windows 10?

Yes. Version 10 Pro 1909.

> You can always access Ditto by the hot key, ctrl-`

Yes. When I restart Ditto, it reappear in the tray.

**Comment by stevemagruder** [comment]
This has been happening to me for a few months or so. It may well be a Windows 10 issue as a few other system tray icons like to disappear unexpectedly. It could also be caused by another errant app, I suppose.

**Comment by JnLlnd** [comment]
In my case, it stopped doing it. I could not say if this was solved by a Windows update but it could be.

**Comment by stevemagruder** [comment]
@JnLlnd are you in the Windows Insider Dev Channel?  I'm in the Beta Channel and haven't gotten a fix yet.

**Comment by JnLlnd** [comment]
No. I'm a regular user. Last update was Version 20H2 (build 19042.928).

And I could not tell it the issue will reappear or not.

**Comment by cgassmann** [comment]
Same issue here on two separate workstations. Currently on Insider Beta with Windows 11 21H2 (build 22000.168), but it started a few months ago on Windows 10 20H2.

**Comment by cgassmann** [comment]
One more observation: Here, the tray icon disappears within a minute after user logon. After restarting Ditto, it doesn't disappear again for the rest of the logon session.

**Comment by stevemagruder** [comment]
This may be a longshot, but I and others have recently realized that the tray icon set up by the TaskbarX app makes other tray icons (particularly for PowerToys) disappear. Does anyone seeing this problem have TaskbarX running on their system?

**Comment by stevemagruder** [comment]
Never mind on my last comment. I just restarted after updating Windows, and the Ditto tray icon is gone. I had to launch Ditto again.  I'm using the beta version 03.24.211.0 64bit.

**Comment by cgassmann** [comment]
The issue seems to have disappeared after updating to Windows 11 build 22000.**194**!

**Comment by stevemagruder** [comment]
After some time, I can say now that with regularity, the Ditto tray icon disappears after any Windows 10 update. I may be upgrading to Win11 soon, so I'll also monitor what happens with that.
